### ISSUE TITLE
Pick a Suitable Color Scheme for Wunderschön

### ISSUE THREAD

**Issue by Maxwell-Alexius** [Maxwell-Alexius/Wunderschon#63]
Some Ideas from Here : https://coolors.co/424b54-7f9296-ffffff-d8759b-a9e5bb

**Comment by Maxwell-Alexius** [comment]
Main color scheme:
```
$outer-space: #424B54;
$roman-silver: #7F9296;
$white: #ffffff;
$pale-chestnut: #E2B4BD;
$sea-foam-greeen: #A9E5BB;

$bootstrap-primary: rgba(2,117,216, 1);
$bootstrap-success: rgb(92, 184, 92);
$bootstrap-danger: rgb(217, 83, 79);
$bootstrap-warning: rgb(240, 173, 78);
```

**Comment by Maxwell-Alexius** [comment]
Closed with PR #77 Merged
